examples/vagrant/Vagrantfile

```diff
@@ -32,6 +32,7 @@ Vagrant.configure("2") do |config|
         vb.cpus = 1
       end
       host.vm.network "private_network", ip: "192.168.100.#{i + 100}"
+      # host.vm.provision "shell", path: 'proxy-only.sh'
```

Comment:
I suppose this is intentionally here, but maybe it could be something like `if ENV['VAGRANT_HTTP_PROXY']`?